config/boards/rk3566-box-demo.csc

```diff
@@ -0,0 +1,33 @@
+# Rockchip RK3566 BoX Demo with 4GB DDR/HDMI/USB3/WIFI/GMAC
```

Comment:
remove "BoX Demo with" an put spaces instead of /. Just compare to other board configuration files.